### PR TITLE
polish tcp authorization user guide

### PR DESCRIPTION
**Pull request by YaoZengzeng**

### CODE REVIEW

**Comment by YaoZengzeng** [comment]
@hzxuzhonghu PTAL

File: content/en/docs/userguide/tcp_authorization.md [hunk 1]
```diff
@@ -103,7 +102,7 @@ This task shows you how to set up authorization policy for TCP traffic in Kmesh.
 - Verify whether requests from the corresponding IP are being allowed.
 
   ```log
-  kubectl exec sleep-78ff5975c6-plh7r -c sleep -- curl  http://httpbin:8000/headers
```
[the remark this answers is not in the document]
Comment (YaoZengzeng) [reply content/en/docs/userguide/tcp_authorization.md:106]: Not sidecar mode, only one container in the pod, no need to specify container with `-c`.

**Comment by YaoZengzeng** [comment]
@hzxuzhonghu updated

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**